mmv1/templates/terraform/examples/cloud_run_service_gpu.tf.erb

```diff
@@ -0,0 +1,34 @@
+resource "google_cloud_run_service" "<%= ctx[:primary_resource_id] %>" {
```

Comment:
We'll need to add an example block to the resource.yaml file to register this example for documentation and test case generation. Please refer to step 5 in https://googlecloudplatform.github.io/magic-modules/develop/test/test/#add-a-create-test for guidance. Additionally, we can remove the handwritten test since the resource configurations in the examples and the handwritten test are mostly identical.

Reply:
Thanks. I have removed the tests in favor of the examples.